<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsMapCanvasDockWidgetBase</class>
 <widget class="QgsDockWidget" name="QgsMapCanvasDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>216</width>
    <height>138</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Map Canvas</string>
  </property>
  <widget class="QWidget" name="mContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QToolBar" name="mToolbar">
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="floatable">
       <bool>false</bool>
      </property>
      <addaction name="mActionSetCrs"/>
      <addaction name="mActionSyncView"/>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="mMainWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="mActionSetCrs">
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/themes/default/propertyicons/CRS.svg</normaloff>:/images/themes/default/propertyicons/CRS.svg</iconset>
   </property>
   <property name="text">
    <string>Set Map CRS</string>
   </property>
   <property name="toolTip">
    <string>Set Map CRS</string>
   </property>
  </action>
  <action name="mActionSyncView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/themes/default/sync_views.svg</normaloff>:/images/themes/default/sync_views.svg</iconset>
   </property>
   <property name="text">
    <string>Synchronize view</string>
   </property>
   <property name="toolTip">
    <string>Synchronize View with Main Map</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsDockWidget</class>
   <extends>QDockWidget</extends>
   <header>qgsdockwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
